### PR TITLE
Use Astra Notices as a composer packages

### PR DESCRIPTION
**Pull request by sushmak02**
### Description
Added Astra notices in the composer

### Screenshots
N/A

### Types of changes

New feature (non-breaking change which adds functionality)

### Checklist:
- [x] My code is tested
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [x] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: lib/astra-notices/class-astra-notices.php [hunk 1]
```diff
@@ -0,0 +1,389 @@
+<?php
```
Comment (vrundakansara) [inline lib/astra-notices/class-astra-notices.php:1]: We have this library at two places - 
https://github.com/brainstormforce/header-footer-elementor/tree/astra-sites-notices/inc/lib/astra-notices
https://github.com/brainstormforce/header-footer-elementor/tree/astra-sites-notices/lib/astra-notices

@sushmak02 - Can you check why?

**Review by vrundakansara — CHANGES_REQUESTED** [review]

File: lib/astra-notices/class-astra-notices.php (hunk 1 above)
Comment (sushmak02) [reply lib/astra-notices/class-astra-notices.php]: Yes, it was due to the wrong path. I just pushed fix for this.

Comment (vrundakansara) [reply lib/astra-notices/class-astra-notices.php]: Perform composer update to update the library. The latest library [here](https://github.com/brainstormforce/astra-notices/releases).
Currently in this repo the library has older version

Comment (sushmak02) [reply lib/astra-notices/class-astra-notices.php]: Done

**Review by vrundakansara — APPROVED** [review]

**Pull request merged**